### PR TITLE
[common] (Requirement Sets) Clean up common requirement set table

### PR DESCRIPTION
**Pull request by AlexJerabek**
[Staging site](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/office-add-in-requirement-sets?branch=AlexJ-ReqSet)

This change breaks up the large table specifying requirement sets for Common API members into smaller subsections. This allows us to link to those subsections from other parts of the documentation.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[29cc46f](https://github.com/OfficeDev/office-js-docs-pr/pull/880/commits/29cc46f86a9574d07649ca1d5a6aa4d539653f9c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/requirement-sets/identity-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-ReqSet/docs/reference/requirement-sets/identity-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/office-add-in-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-ReqSet/docs/reference/requirement-sets/office-add-in-requirement-sets.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C9%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904092240528535-880%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=TSj%2F%2FmROAHZbqWcBIp06QGS9IUumdIZGiRVkB3P9RAA%3D&st=2019-04-09T22%3A37%3A03Z&se=2019-05-10T22%3A42%3A03Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[f16919d](https://github.com/OfficeDev/office-js-docs-pr/pull/880/commits/f16919d25c72ad3f206116ad0235cd19d71d8d50)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/requirement-sets/identity-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-ReqSet/docs/reference/requirement-sets/identity-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/office-add-in-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-ReqSet/docs/reference/requirement-sets/office-add-in-requirement-sets.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C9%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904092241228581-880%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=xTnZiZ0R7kmHH%2B5e9OWoKliY%2BnjynmmxAezSJDwsiNo%3D&st=2019-04-09T22%3A38%3A26Z&se=2019-05-10T22%3A43%3A26Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[987f590](https://github.com/OfficeDev/office-js-docs-pr/pull/880/commits/987f590cb29dd333485f81a0c6bb19e6f5a9a9fb)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/requirement-sets/identity-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-ReqSet/docs/reference/requirement-sets/identity-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/office-add-in-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-ReqSet/docs/reference/requirement-sets/office-add-in-requirement-sets.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C9%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904092242506057-880%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=7%2F4lJt1%2Fobpp9kAIsNBPjzvDYX5hHrlhpyAn1Fi1%2BKE%3D&st=2019-04-09T22%3A39%3A37Z&se=2019-05-10T22%3A44%3A37Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/requirement-sets/office-add-in-requirement-sets.md [hunk 1]
```diff
@@ -13,45 +13,208 @@ Requirement sets are named groups of API members. Office Add-ins use requirement
...
-| TextCoercion  | Excel<br>Excel Online<br>Excel for iPad<br>OneNote Online<br>PowerPoint<br>PowerPoint Online<br>PowerPoint for iPad<br>PowerPoint for Mac<br>Project<br>Word 2013 and later<br>Word 2016 for Mac and later<br>Word Online<br>Word for iPad|Supports coercion to text format (Office.CoercionType.Text) when reading and writing data using the Document.getSelectedDataAsync, Document.setSelectedDataAsync, Binding.getDataAsync, or Binding.setDataAsync methods.|
-| TextFile  | Word 2013 and later<br>Word 2016 for Mac and later<br>Word Online<br>Word for iPad|Supports output to text format (Office.FileType.Text) when using the Document.getFileAsync method.|
+The following section list the Common API requirement sets, the methods in each set, and the Office host applications that support that requirement set. All of these API requirement sets are version 1.1.
```
Comment (ElizabethSamuel-MSFT) [inline docs/reference/requirement-sets/office-add-in-requirement-sets.md]: Perhaps "The following sections list..."

**Comment by o365devx** [comment]
Docs Build status updates of commit _[8d256ea](https://github.com/OfficeDev/office-js-docs-pr/pull/880/commits/8d256ea75e52d1930285b0d8938f8bd52fbaf6c2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/requirement-sets/identity-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-ReqSet/docs/reference/requirement-sets/identity-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/office-add-in-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-ReqSet/docs/reference/requirement-sets/office-add-in-requirement-sets.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C9%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904092252091203-880%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ICGpxO2SF1x9NyCWaz7dpZyN9NHGHHPdLryPjGygHog%3D&st=2019-04-09T22%3A48%3A17Z&se=2019-05-10T22%3A53%3A17Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**